README.md

```diff
@@ -1,52 +1,61 @@
-## Welcome to OpenEBS
-[![OpenEBS Welcome Banner](https://github.com/openebs/community/blob/develop/images/waterfalls-Gamer-16-bit-pixel-art_HERO_banner.png)](https://www.openebs.io/)
+## Project Purpose
 
-OpenEBS is a modern Block-Mode storage platform, a Hyper-Converged software Storage System and virtual NVMe-oF SAN (vSAN) Fabric that is natively integrates into the core of Kubernetes.<BR>
+OpenEBS is an open-source storage service for Kubernetes applications. OpenEBS manages the block storage and file systems based on the block storage for containers running in Kubernetes. Use OpenEBS for creating fast and resilient storage; with options for single-node, and replicated multi-node storage.<BR>
 
 | [<img src="https://github.com/openebs/community/blob/develop/images/slack_icon_small.png" width="100">](https://kubernetes.slack.com/messages/openebs)  | **Try our Slack channel** <BR>If you have questions about using OpenEBS, please use the CNCF Kubernetes **OpenEBS slack channel**, it is open for [anyone to ask a question](https://kubernetes.slack.com/messages/openebs/) <BR> |
 | :---         | :---      |
 
 <BR>
 
 ## Monthly Community Meetings
-TThe community meetings are usually held monthly on the last thursday of the month at 14:00 UTC.<BR>
-The meeting is currently held on Zoom: https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1<BR>
-Upcoming meeting: 29 Aug 2024 at 14:00 UTC
+OpenEBS holds a monthly community meeting via Zoom on the last Thursday of the month, at 7 am Pacific time.
+<br>
+The next meeting is on: `Thursday 26 September, at 14:00 UTC`
```

Comment:
This is difficult to maintain. Maybe the meeting link is enough?
The mention of Pacific time and UTC is rather confusing. Can we stick with either of the two?

Reply:
Also, with Pacific time, the exact time is sometimes difficult to tell when daylight savings is on. My recommendation would be UTC.

Reply:
Sure. I would change it to UTC.